simpunch/level2.py

```diff
@@ -135,22 +138,98 @@ def generate_starfield(wcs: WCS,
     return fake_image, sources
 
 
-def add_starfield_polarized(input_data: NDCube) -> NDCube:
-    """Add synthetic starfield."""
+def generate_dummy_polarization(map_scale: float = 0.225,
```

Comment:
Could you upload an image of this map to the review just so we can confirm it? Please include a colorbar.

Reply:
![Uploading Screenshot 2024-11-24 at 12.12.23.png…]()

Reply:
<img width="1741" alt="Screenshot 2024-11-24 at 12 12 23" src="https://github.com/user-attachments/assets/ffc4a462-a4cd-4664-8acb-16b4ca24d103">